docs/troubleshooting.md

```diff
@@ -46,7 +46,7 @@ kubectl get secrets wavefront-secret -n observability-system -o json | jq '.data
 #### Unknown host or Unable to check in
 - *Without an HTTP Proxy*
 
-  - Check that you configured the correct Wavefront URL in your Wavefront CR
+  - Check that you configured the correct Wavefront URL in your Observability for Kubernetes Operator Custom Resource
```

Comment:
Wavefront URL --> Operations for Applications URL

Reply:
We don't think we should change this one since the actual property is called wavefrontURL. I think we should also revert this original change we made too, since the custom resource is still called "Wavefront" (and we can't really change that).